packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -170,12 +170,15 @@ export class PaymentChannelClient {
   }
 
   insertIntoChannelCache(channelState: ChannelState) {
-    this.channelCache = {...this.channelCache, [channelState.channelId]: channelState};
+    // TODO: The joinChannel promise never resolves, meaning this function is not called
```

Comment:
Is this still the case? I thought we fixed the `joinChannel` not resolving but I could be wrong.

Reply:
It's an out of date comment, thanks to a three-way merge